src/store/slicePopularCategory/slicePopularCategories.ts

```diff
@@ -0,0 +1,54 @@
+import {createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { LOAD_STATUSES } from '../../constants'
+import {Category, Good,Api } from "api/api";
```

Comment:
реэкспорт через index
```suggestion
import {Category, Good,Api } from "api";
```